hoot-core-test/src/test/cpp/hoot/core/ops/MapCropperTest.cpp

```diff
@@ -331,39 +333,49 @@ class MapCropperTest : public HootTestFixture
     uut.setKeepEntireFeaturesCrossingBounds(false);
     uut.setKeepOnlyFeaturesInsideBounds(true);
     uut.apply(map);
-//    LOG_WARN("runKeepFeaturesOnlyCompletelyInBoundsTest-2");
-//    LOG_VARW(map->getNodes().size());
-//    LOG_VARW(map->getWays().size());
-//    MapProjector::projectToWgs84(map);
-//    testFileName = "runKeepFeaturesOnlyCompletelyInBoundsTest-2.osm";
-//    OsmMapWriterFactory::write(map, _outputPath + "/" + testFileName);
-    CPPUNIT_ASSERT_EQUAL(7, (int)map->getNodes().size());
-    CPPUNIT_ASSERT_EQUAL(1, (int)map->getWays().size());
-
-    // setting invert to true negates the keep the keep only features inside bounds setting;
+    MapProjector::projectToWgs84(map);
```

Comment:
Lots of new tests here to deal with the options added to `MapCropper`, which either allow you to keep features crossing the bounds (like the way API DB queries work) or only keep features
that reside entirely within the bbox. The default behavior of `MapCropper` is still to split features that cross the bounds as it always has been.